packages/client/hmi-client/src/services/intervention-policy.ts

```diff
@@ -17,6 +17,8 @@ export const getInterventionPolicyById = async (policyId: string): Promise<Inter
 
 export const createInterventionPolicy = async (policy: InterventionPolicy): Promise<InterventionPolicy> => {
 	delete policy.id;
+	delete policy.createdOn;
+	delete policy.updatedOn;
```

Comment:
How do we get a createdOn/updatedOn fields before we actually created the object?

Reply:
We don't, but since they don't exist nothing will happen.  This is more for the case when we want to create a new intervention policy with "save as."  I pass in the whole intervention policy which does have those keys.

Reply:
Ideally the API should just ignore, or overwrite, any extra fields it doesn't use. 
I am guessing this is related to https://github.com/DARPA-ASKEM/terarium/issues/4488 so feel free to add pain-points on that ticket.